libs/docs/core/dynamic-page/examples/dynamic-page-dynamic-container-height-example/dynamic-page-dynamic-container-height-example.component.html

```diff
@@ -6,6 +6,7 @@
         }
         <fd-dynamic-page size="large" ariaLabel="Example Dynamic Page" [autoResponsive]="false">
             <fd-dynamic-page-header
+                headingLevel="h1"
```

Comment:
dynamic page heading should be level 2. Level 1 is usually reserved for the shellbar title.